src/test/java/hudson/plugins/jira/deprecated/DeprecatedJiraBuildAction.java

```diff
@@ -0,0 +1,57 @@
+package hudson.plugins.jira.deprecated;
+
+import hudson.model.AbstractBuild;
+import hudson.model.Action;
+import hudson.plugins.jira.JiraIssue;
+import hudson.plugins.jira.Messages;
+
+import java.util.*;
+
+/**
+ * JIRA issues related to the build.
```

Comment:
Needs a description on why this class is here and what do the corresponding tests test in fact.
Also, please change or remove @author.